### DISCUSSION TITLE
WebApi Route to Logout with a POST Request from a micro controller

### DISCUSSION THREAD

**Discussion by Thom087** [pocketbase/pocketbase#2823] [Q&A] [answered]
Hi 
I want to login with a micro controller over a post request.
This can be done as follows: POST /api/admins/auth-with-password
But how can I logout with a POST request?

Thanks in advance.

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase is fully stateless, meaning that there are no sessions in the traditional sense, neither the tokens are stored in the database. An admin or user is considered authenticated if a request is sent with `Authorization: TOKEN` header.

Previously issued JWT tokens are automatically invalidated after they expire, when the account reset their password, or when changing the common admin token secret (the last one affect all admins).

**Reply by Thom087** [reply]
Ok thanks
